docs/external-registry-ca.md

```diff
@@ -17,7 +17,7 @@ kind: ConfigMap
 metadata:
   name: trusted-ca
 data:
-  cincinnati-registry: |
+  updateservice-registry: |
```

Comment:
Not sure if its needed. IIUC this key depends on the CR name, @wking?

Reply:
From observation it is needed, https://github.com/openshift/cincinnati-operator/blob/a3107f885a3290f10ae24147c27930bbc911f1a0/controllers/names.go#L19-L20